### ISSUE TITLE
Secure cookies are not working

### ISSUE THREAD

**Issue by yoannjly** [yifeikong/curl_cffi#145]
**Describe the bug**
I need to use [Secure Cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#cookie-namecookie-value) to scrape some data.
When I use a synchronous Session and add the cookie using kwargs cookies={name: value}, it does not work, but it works when I add it using a curl-like API.
For AsyncSession, it does not work when I add it using kwargs and the curl-like API.

**Expected behavior**
It should always print 'cookie in headers'.

**Versions**
 - OS: Windows 11 | Alpine Linux
 - curl_cffi: 0.5.9 | 0.5.10b2
 - python: 3.11

**To Reproduce**
```python
import asyncio
import sys

from curl_cffi import CurlOpt
from curl_cffi.requests import AsyncSession, Session

value = "1"
name = "__Secure-test-cookie"  # will only work for 'bypass test' (request is not sent with the cookie, not https?)
# name = "_Secure-test-cookie"  # will work for all except 'async bypass test' (acurl.setopt() bug?)
cookie = f"{name}={value}"

cookies_bytes = bytes(cookie, 'utf-8')
cookies_dict = {name: value}


def test():
    print("\ntest")
    with Session() as s:
        resp_ = s.get("https://browserleaks.com/ip", impersonate="chrome110", cookies=cookies_dict)
        check_in_headers(resp_.text)


def bypass_test():
    print("\nbypass test")
    with Session() as s:
        s.curl.setopt(CurlOpt.COOKIE, cookies_bytes)
        resp_ = s.get("https://browserleaks.com/ip", impersonate="chrome110")
        check_in_headers(resp_.text)


async def async_test():
    print("\nasync bypass test")
    async with AsyncSession() as s:
        resp_ = await s.get("https://browserleaks.com/ip", impersonate="chrome110", cookies=cookies_dict)
        check_in_headers(resp_.text)


async def async_bypass_test():
    print("\nasync bypass test")
    async with AsyncSession() as s:
        s.acurl.setopt(CurlOpt.COOKIE, cookies_bytes)
        resp_ = await s.get("https://browserleaks.com/ip", impersonate="chrome110")
        check_in_headers(resp_.text)


def check_in_headers(html):
    if cookie not in html:
        print("cookie not in headers")
    else:
        print("cookie in headers")


if sys.version_info >= (3, 8) and sys.platform.lower().startswith("win"):
    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())

test()
bypass_test()
asyncio.run(async_test())
asyncio.run(async_bypass_test())
```

**Comment by yifeikong** [comment]
For `__Secure-*` cookies, `Secure` attribute must be set to True, and that's where we missed in the current implementation. I will change this behavior in the next version.

**Comment by yifeikong** [comment]
Fixed in 0.5.10b3
